extension/src/plots/model/index.ts

```diff
@@ -227,7 +228,11 @@ export class PlotsModel extends ModelWithPersistence {
     return selectedRevisions
   }
 
-  public savePlotData(plotId: string, filePath: string) {
+  public async savePlotData(
```

Comment:
[I] Rather than have a flag argument for `savePlotData` I would have different functions `savePlotDataAsJson`, etc. Even if the `write` functions are passed to a base function.

Reply:
that would then give different messages from the webview which could have different telemetry events and tests.

Reply:
Took off the flag argument and used separate functions instead!